flipperdevices/flipperzero-firmware
# [FL-2932] TikTok: reset cursor after enter and reconnect

# What's new

- Reset cursor position on tiktok view enter and connect event

# Verification 

- Open ble hid app. Connect to phone. Open tiktok controller. Verify it works fine.
- Open ble hid app. Open tiktok controller. Connect to phone. Verify it works fine.

# Checklist (For Reviewer)
